miner/app/src/compute_horde_miner/miner/miner_consumer/validator_interface.py

```diff
@@ -7,7 +7,10 @@
 
 import bittensor
 from compute_horde.mv_protocol import miner_requests, validator_requests
-from compute_horde.mv_protocol.validator_requests import BaseValidatorRequest
+from compute_horde.mv_protocol.models import JobFinishedReceipt, JobStartedReceipt
+from compute_horde.mv_protocol.validator_requests import (
+    BaseValidatorRequest,
+)
```

Comment:
Does ruff not like simple formatting? :|
```suggestion
from compute_horde.mv_protocol.validator_requests import BaseValidatorRequest
```